sauce/features/budget/resize-inspector/index.js

```diff
@@ -1,8 +1,9 @@
-import { Feature } from 'core/feature';
-import * as toolkitHelper from 'helpers/toolkit';
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
```

Comment:
I believe this was fixed in this commit: https://github.com/toolkit-for-ynab/toolkit-for-ynab/commit/7d92a04e3ab23b91830957cab4e259f4b9ae6772

Might need to pull from upstream.